src/functionalTest/resources/application.yaml

```diff
@@ -16,8 +16,9 @@ azure:
 
 bulk-scan-procesor:
   api:
-    retries: 10
-    wait-time-in-ms: 1000
+    url: ${PROCESSOR_URL}
+    retries: 5
+    wait-time-in-ms: 10
```

Comment:
why do we need this?

Reply:
Removed it after adding ignore functional profile.